### DISCUSSION TITLE
I did an installation from scratch with laravel 10.43 and installed livewire 3 and then installed powergrid 5

### DISCUSSION THREAD

**Discussion by LeandroAndrade2020** [Power-Components/livewire-powergrid#1371] [Q&A] [answered]
I've done different installations and always getting this error:

Method PowerComponents\LivewirePowerGrid\PowerGridFields::addColumn does not exist.

And showing that it is in this part of the code:

    public function fields(): PowerGridFields

    {

        return PowerGrid::fields()

            ->addColumn('id')

            ->addColumn('name')

            ->addColumn('email')

            ->addColumn('created_at_formatted', fn (User $model) => Carbon::parse($model->created_at)->format('d/m/Y H:i:s'));

    }
![Captura de tela de 2024-02-02 17-06-31](https://github.com/Power-Components/livewire-powergrid/assets/62396904/07ccc010-1948-4315-a84d-0aa413eab8e6)

**Comment by dansysanalyst** [comment, marked as answer]
Hi @LeandroAndrade2020, thanks for letting us. The stubs were fixed.

Please use `add(...)` instead of `addColumn(..)`.

```php
use PowerComponents\LivewirePowerGrid\PowerGrid;
use PowerComponents\LivewirePowerGrid\PowerGridFields;

public function fields(): PowerGridFields
{
    return PowerGrid::fields()
        ->add('id')
        ->add('name')
        ->add('email')
        ->add('created_at_formatted', fn (User $model) => Carbon::parse($model->created_at)->format('d/m/Y H:i:s'));
}
```

**Comment by luanfreitasdev** [comment]
Fixed https://github.com/Power-Components/livewire-powergrid/releases/tag/v5.3.4
